lib/ruleHelper.js

```diff
@@ -117,8 +127,9 @@ RuleHelper.prototype = {
 
     /**
      * Returns functionName or objectName.methodName of an expression
-     * @param {Object} node
-     * @returns {String} nice name to expression call
+     *
+     * @param {Object} node This is turned into string to look at a callable
```

Comment:
Typo: *This is turned into **a** string to look at a callable*

But I don't think this describes the param. Perhaps *Call expression node* is fine like `shouldCheckMethodCall` has.